populate_db.py

```diff
@@ -74,12 +74,28 @@ def create_settings():
         stripe_secret_key = env('STRIPE_SECRET_KEY', default=None)
         stripe_publishable_key = env('STRIPE_PUBLISHABLE_KEY', default=None)
         stripe_client_id = env('STRIPE_CLIENT_ID', default=None)
+        fb_client_id = env('FACEBOOK_CLIENT_ID', default=None)
+        fb_client_secret = env('FACEBOOK_CLIENT_SECRET', default=None)
+        google_client_id = env('GOOGLE_CLIENT_ID', default=None)
+        google_client_secret = env('GOOGLE_CLIENT_SECRET', default=None)
+        tw_consumer_key = env('TWITTER_CONSUMER_KEY', default=None)
+        tw_consumer_secret = env('TWITTER_CONSUMER_SECRET', default=None)
+        in_client_id = env('INSTAGRAM_CLIENT_ID', default=None)
+        in_client_secret = env('INSTAGRAM_CLIENT_SECRET', default=None)
 
         if stripe_client_id and stripe_secret_key and stripe_publishable_key:
```

Comment:
You can remove this condition. Right now it won't add fb/google etc. client ids if stripe id is none. It was a redundant condition added by me. Please remove this.